LiteLoader/Kernel/EventAPI.cpp

```diff
@@ -1952,4 +1953,18 @@ THook(void, "?dropSlot@Inventory@@QEAAXH_N00@Z", Container* a1, int a2, char a3,
         IF_LISTENED_END(PlayerDropItemEvent)
     }
     return original(a1, a2, a3,a4,a5);
-}
\ No newline at end of file
+}
+
+THook(char, "?take@Player@@QEAA_NAEAVActor@@HH@Z", Player* a1, Actor& a2, int a3, int a4)
```

Comment:
```suggestion
THook(char, "?take@Player@@QEAA_NAEAVActor@@HH@Z", Player* a1, Actor& a2, int a3, int a4)
```
Please use `TInstanceHook` like above